cypress/src/tests/b2b/verifyPurchaseOrders.spec.js

```diff
@@ -0,0 +1,663 @@
+/**
+ * B2B Purchase Orders Test Suite
+ *
+ * This test suite verifies the Purchase Orders functionality for B2B customers.
+ * Tests include creating, approving, and managing purchase orders.
+ */
+
+import { createUserAssignCompanyAndRole } from "../../support/b2bPOAPICalls";
+
+describe("B2B Purchase Orders", () => {
+  const urls = {
+    login: "/customer/login",
+    account: "/customer/account",
+    product: "/products/adobe-pattern-hoodie/adb127",
+    cheepProduct: "/products/badge-reel/adb153",
+    checkout: "/checkout",
+    purchaseOrders: "/customer/purchase-orders",
+    approvalRules: "/customer/approval-rules",
+  };
+
+  const selectors = {
+    loginForm: "main .auth-sign-in-form",
+    emailInput: 'input[name="email"]',
+    passwordInput: 'input[name="password"]',
+    submitButton: 'button[type="submit"]',
+    addToCartButton: "button",
+    navCartButton: ".nav-cart-button",
+    checkoutLink: 'a[href="/checkout"]',
+    checkMoneyOrderLabel: "label",
+    termsCheckbox: 'input[type="checkbox"]',
+    placePOButton: 'button[type="button"]',
+    navDropdownButton: ".nav-dropdown-button",
+    logoutButton: "button",
+    checkbox: 'input[type="checkbox"]',
+    companyPOContainer: '[data-testid="company-purchase-orders-container"]',
+    approvalPOWrapper:
+      ".commerce-b2b-po-require-approval-purchase-orders-wrapper",
+    myApprovalPOWrapper: ".commerce-b2b-po-require-approval-purchase-orders",
+    poTable: ".b2b-purchase-order-purchase-orders-table",
+    showButton: "button",
+    editButton: "button.dropin-action-button",
+    nameInput: 'input[name="name"]',
+    statusCheckbox: 'input[type="checkbox"][name="status"]',
+    textarea: "textarea",
+    ruleTypeSelect: 'select[name="ruleType"]',
+    ruleConditionSelect: 'select[name="ruleCondition"]',
+    ruleValueInput: 'input[name="ruleValue"]',
+    multiSelect: ".dropin-multi-select",
+  };
```

Comment:
@KostiantynFandeliuk Please move these to https://github.com/hlxsites/aem-boilerplate-commerce/blob/po-cypress/cypress/src/fields/index.js